.github/workflows/test-build-deb-temp.yml

```diff
@@ -0,0 +1,422 @@
+name: Process Debian Artifacts
+
+on: 
+#  schedule:
+#    - cron: '0 10 * * *'
+
+  repository_dispatch:
+    types: [build-deb]
+
+jobs:
+  plugin-availability:
+    name: Check Plugin Availability
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Send Dispatch Event for check_plugin Workflow
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.GitHub_PAT }}
+          event-type: check_plugin_temp # Require Changes
+          client-payload: '{"PLUGIN_TYPES_PL": "deb kibana", "ODFE_VERSION_PL": "", "CHIME_ALERT_PL": "true", "EMAIL_ALERT_PL": "true"}'
+
+      - name: Sleep and wait for check_plugin to trigger
+        uses: juliangruber/sleep-action@v1
+        with:
+          time: 20s
+
+      - name: Wait for check_plugin Workflow Status Return
+        uses: lewagon/wait-on-check-action@master
+        id: wait-for-check_plugin
+        with:
+          repo-token: ${{ secrets.GitHub_PAT }}
+          checki-name: Check Plugin Availability Main # Require Changes
```

Comment:
If there are concurrent running for this check plugin, how should we differentiate it?

Reply:
Add concurrent checks.

Example runs here, all 3 jobs triggers with 1 second apart:
https://github.com/peterzhuamazon/opendistro-build/actions/runs/112846281
![image](https://user-images.githubusercontent.com/65193828/82715978-503b9980-9c63-11ea-8488-2eab9d5d0d5b.png)

https://github.com/peterzhuamazon/opendistro-build/actions/runs/112846290
![image](https://user-images.githubusercontent.com/65193828/82715989-5e89b580-9c63-11ea-8ee7-c26ac9e3cd86.png)

https://github.com/peterzhuamazon/opendistro-build/actions/runs/112846318
![image](https://user-images.githubusercontent.com/65193828/82716008-76613980-9c63-11ea-9de4-c4f6781283d8.png)